src/generated/cloud/lustre/v1/Cargo.toml

```diff
@@ -0,0 +1,47 @@
+# Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Code generated by sidekick. DO NOT EDIT.
+
+[package]
+name                   = "google-cloud-lustre-v1"
+version                = "0.3.0"
```

Comment:
note for the future: when we are at 1.0, would we want to create these as 1.0?

Reply:
I think so. We expect them to be stable and just work.

Reply:
Sorry, I missed your comment about this not being `"release_level": "GA"`. We probably aren't allowed to make it `1.0`.